ui/src/cells/actions/thunks.ts

```diff
@@ -0,0 +1,162 @@
+// Libraries
+import {normalize} from 'normalizr'
+
+// APIs
+import * as api from 'src/client'
+import * as dashAPI from 'src/dashboards/apis'
+
+// Schemas
+import * as schemas from 'src/schemas'
+
+// Actions
+import {refreshDashboardVariableValues} from 'src/dashboards/actions/thunks'
+import {setView} from 'src/dashboards/actions/views'
+import {notify} from 'src/shared/actions/notifications'
+import {setCells, setCell, removeCell} from 'src/cells/actions/creators'
+
+// Utils
+import {getClonedDashboardCell} from 'src/dashboards/utils/cellGetters'
+
+// Constants
+import * as copy from 'src/shared/copy/notifications'
+
+// Types
+import {
+  Dashboard,
+  NewView,
+  Cell,
+  GetState,
+  RemoteDataState,
+  NewCell,
+  DashboardEntities,
+  ResourceType,
+  CellEntities,
+} from 'src/types'
```

Comment:
wait... and you were explicit on types?! which are removed from the dependency graph?!